srcalc/src/main/java/gov/va/med/srcalc/domain/model/AbstractVariable.java

```diff
@@ -157,15 +161,51 @@ public final void setGroup(final VariableGroup group)
         fGroup = Objects.requireNonNull(group, "group must not be null");
     }
 
+    /**
+     * Returns the help text as a String, not an Optional. Intended for
+     * Hibernate use.
+     * @return the help text String if present, otherwise null.
+     */
     @Basic
-    public final String getHelpText()
+    @Column(
+            name = "help_text",
+            nullable = true,
+            length = Variable.HELP_TEXT_MAX)
+    final String getHelpTextString()
+    {
+        return fHelpText.orNull();
+    }
+    
+    /**
+     * Sets the help text as a nullable String. Intended for Hibernate use.
+     * @param helpText may be null
+     */
+    public final void setHelpTextString(final String helpText)
+    {
+        setHelpText(Optional.fromNullable(helpText));
+    }
+
+    @Transient // getHelpTextString() is mapped instead
+    public final Optional<String> getHelpText()
     {
         return fHelpText;
     }
 
-    public final void setHelpText(final String helpText)
+    /**
+     * Sets the help text. See {@link #getHelpText()}.
+     * @throws NullPointerException if the given object is null
+     * @throws IllegalArgumentException if the given help text is longer than
+     * {@link Variable#HELP_TEXT_MAX} characters or an empty String. (The
+     * absence of a help text must be represented as an "absent" Optional, not
+     * an empty string.)
+     */
+    public final void setHelpText(final Optional<String> helpText)
     {
-        this.fHelpText = helpText;
+        if (helpText.isPresent())
+        {
+            Preconditions.requireWithin(helpText.get(), 1, HELP_TEXT_MAX);
```

Comment:
If we are allowing null, then why would it matter if we had a 0 character String?